helix-term/src/commands.rs

```diff
@@ -3509,12 +3509,22 @@ fn open(cx: &mut Context, open: Open) {
 
         let indent_len = indent.len();
         let mut text = String::with_capacity(1 + indent_len);
-        text.push_str(doc.line_ending.as_str());
-        text.push_str(&indent);
 
-        if let Some(token) = continue_comment_token {
-            text.push_str(token);
-            text.push(' ');
+        if new_line == 0 {
```

Comment:
I think this is correct but please add an explatory comment, I needed to think about this to a second (meaning it's entirely opaque to anybody new to the codebase). This is caused by:

* `let line_num = new_line.saturating_sub(1) == new_line` for `new_line=0`
* can only occur for open_above (because for open_below new_line >= 1)

Reply:
Ok, I renamed some variables (and also removed an unused variable) which should make that more readable (at least in my opinion).
If it doesn't help, I'll try to add a bigger comment (and revert my renamings).